mpas_analysis/shared/io/mpas_reader.py

```diff
@@ -195,8 +195,9 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
         timeStrings = [''.join(xtime.astype('U')).strip()
                        for xtime in timeVar.values]
         for i, timeString in enumerate(timeStrings):
-             if timeString[8:] == '15_00:00:00' and i +1 < len(timeStrings):
-                 timeStrings[i] = f'{timeStrings[i + 1][:7]}-01_00:00:00'
+             if (timeString == '0000-01-15_00:00:00' and
+                 i + 1 < len(timeStrings)):
+                     timeStrings[i] = f'{timeStrings[i + 1][:7]}-01_00:00:00'
```

Comment:
Maybe not super intuitive but this is the preferred python formatting:
```suggestion
             if (timeString == '0000-01-15_00:00:00' and
                     i + 1 < len(timeStrings)):
                 timeStrings[i] = f'{timeStrings[i + 1][:7]}-01_00:00:00'
```